src/CMakeLists.txt

```diff
@@ -298,24 +298,48 @@ endif()
 # Installation
 #
 
```

Comment:
can we also migrate to https://github.com/openvinotoolkit/openvino/pull/24657/files#diff-b3f6c5638eca5dc9293d5f78fd6418672c783184b2d86a9485d9547727558f1aR485 ?